CSS/ex_16/about.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About | It's Just Jack</title>
+    <style>
+        @import url('https://fonts.googleapis.com/css2?family=Homemade+Apple&display=swap');
+        header {
+            text-align: center ;
+            color:rgb(241, 77, 55) ;
+        }
+        h1 {
+            text-align: center;
+        }
+
+        h2 {
+            font-size: 30px ;
+            font-family: 'Homemade Apple', cursive ;
+        }
+        
+        /* Flexbox navigation goes below here */
```

Comment:
Well done placing comments in CSS! It gives better understanding of which css does what.
It is also nice and concise without unnecessary comments.
Good job!